### ISSUE TITLE
Train own datasets

### ISSUE THREAD

**Issue by wakeupppp** [zhanghang1989/PyTorch-Encoding#320]
1. 您好，我在尝试构建自己的VOC数据集并利用encnet训练，只修改了`pascal_voc.py`，如下
![image](https://user-images.githubusercontent.com/39750646/91250978-e4e6f380-e78c-11ea-9e9b-4409caacce8d.png)
但是报如下的错误
![image](https://user-images.githubusercontent.com/39750646/91251031-0e078400-e78d-11ea-8a3c-6ebf0358a7e2.png)
2. 于是我又尝试利用原有的`pascal_voc`数据集进行训练，21类时一切正常，但是在`pascal_voc.py`中减少一类时，会发生如下错误
![image](https://user-images.githubusercontent.com/39750646/91251314-b9183d80-e78d-11ea-94ec-09f91f4fc926.png)

![image](https://user-images.githubusercontent.com/39750646/91251289-adc51200-e78d-11ea-8c45-0a25907cd459.png)
3. CUDA是10.2版本，其余环境如下，在单个GPU上训练，命令是`python train_dist.py --dataset Pascal_voc --model encnet   --se-loss --backbone resnet101 --lr 0.001 --export rock --ft`，不知我是否应该改漏了哪些地方，万分感谢！
![image](https://user-images.githubusercontent.com/39750646/91251451-144a3000-e78e-11ea-9fca-dc238b62703c.png)

**Comment by zhanghang1989** [comment]
问题1，不太清楚，可能你要检查一下数据集

问题2，这样改是不行的，因为 data 没有改，实际运行的时候还是21 类，这样在取 index 求 softmax crossentropy loss 的时候就挂了

3. 应该不是环境的问题

**Comment by wakeupppp** [comment]
> 问题1，不太清楚，可能你要检查一下数据集
> 
> 问题2，这样改是不行的，因为 data 没有改，实际运行的时候还是21 类，这样在取 index 求 softmax crossentropy loss 的时候就挂了
> 
> 1. 应该不是环境的问题

十分感谢！经检查，发现自己的数据集是RGB三通道24位，VOC中png默认是8位，处理完毕即可

**Comment by zhanghang1989** [comment]
👍
